Remove if/else/unless from booleans concept. (#2660)
exercism/v3: languages/ruby/exercises/concept/booleans/.docs/introduction.md

> ## Control flow

Reviewer: ```suggestion
```

> ## Control flow
> 

Reviewer: ```suggestion
```

> _Truthy_ and _falsey_ evaluations are useful in the context of control flow.

Reviewer: [Lines 33-54](https://github.com/exercism/v3/blame/83b6a40174d0db1ff3fb4681c00eb860a8be1219/languages/ruby/exercises/concept/booleans/.docs/introduction.md#L33-L54) still contain if and unless

Author: @neenjaw So we don't want nothing connected to `if/else` here? Yup?

Reviewer: Yeppers, so it makes it kind of tricky.  All we need to give is _just enough_ information for the student to go on so that they can google it further.  And since `if/else/unless` aren't required for this exercise, they should be removed.

:tada:

> _Truthy_ and _falsey_ evaluations are useful in the context of control flow.

Reviewer: This statement should be revised now that those examples are gone and there are no control flow examples.  Maybe a statement on how truthy and falsey are defined, with the remaining code example?

Author: What's about adding sample with `!!` operator. We could show that we can use `!!` to determine truthiness of object.

Reviewer 2: This isn't really the case, we use this to coerce to Boolean, rather than determining the truthiness of the object.

Reviewer: ```suggestion
In ruby, all values can be evaluated as _truthy_ or _falsey_. When a value is used in a boolean context, its _truthy_ or _falsey_ value is used to evaluate the boolean expression.  Only `nil` and `false` are _falsey_, all other values are _truthy_.
```
